src/test/java/com/segment/analytics/android/integrations/amplitude/AmplitudeTest.java

```diff
@@ -606,6 +642,30 @@ private void verifyAmplitudeLoggedEvent(String event, JSONObject jsonObject) {
     verify(amplitude).logEvent(eq(event), jsonEq(jsonObject), isNull(JSONObject.class), eq(false));
   }
 
+  private void verifyOutOfSession(String event, JSONObject jsonObject, @Nullable Map options) {
```

Comment:
There are a lot of cases in here, there's a chance the logic in this test helper can break  and could cause false positives.

I think it's better to have this just be inlined inside the functions. It doesn't seem to me that this saves that many lines of code anyway.

It would also be good to not call reset (they should be different test methods).

Reply:
Gotcha - I totally misunderstood your previous comment about the helper function. Sorry.